app/services/backend_service.rb

```diff
@@ -0,0 +1,25 @@
+class BackendService
+  def self.conn
+    Faraday.new(url: ENV["BACKEND_URL"])
+  end
+
+  def self.flights_search(query)
+    response = conn.get('/flight_search') do |req|
```

Comment:
We'll want to change this conn.get call to use the argument ('/flights') to match the path in the Flight-API microservice.

Reply:
Since this is the frontend, the backend will have to manipulate the data, so it wouldn't exactly make sense to just pass everything through to the frontend.

Reply:
We updated this to be /flights